src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.Windows/WinRT/Windows.Ui.Notifications.Generated.cs

```diff
@@ -0,0 +1,3612 @@
+#pragma warning disable 108
```

Comment:
This file doesn't need to be generated.
You also can use SourceGen for MicroCom:
- https://www.nuget.org/packages/MicroCom.CodeGenerator.Roslyn
- rename ".idl" to ".mcidl"
- Include it with [AdditionalFiles](https://github.com/dotnet/roslyn/blob/main/docs/analyzers/Using%20Additional%20Files.md)

Reply:
Done.